code/modules/mob/living/carbon/human/human_attackhand.dm

```diff
@@ -199,7 +199,10 @@
 				G.last_action = world.time
 				visible_message("<span class='warning'>[M] gets a strong grip on [src]!</span>")
 				return 1
-			visible_message("<span class='warning'>[M] has grabbed [src] passively!</span>")
+			if(src.has_tail() && H.has_tail())
+				H.visible_message(SPAN_WARNING("[H] coils their tail with [src], wow is that okay in public?!"), "You entwine your tail with [src]'s!")
```

Comment:
```suggestion
				var/in_public = length(viewers(H)+viewers(src)) > 3
				H.visible_message(span(in_public ? "danger" : "warning, "[H] coils their tail with [src]'." + (in_public) ? "Wow, is that okay in public?!" : ""), SPAN_NOTICE("You entwine your tail with [src]'s!"))
```

Reply:
it's outdated i can't merge it but @oranges you should backport this

Reply:
oh fuck it pinged a random oranges

Reply:
@optimumtact high priority fix

Reply:
thanks I'll use this